.github/workflows/ci-on-push.yml

```diff
@@ -0,0 +1,71 @@
+name: Continuous integration
+
+on: push
+
+jobs:
+  test_lint_deploy:
+    name: 'Test, lint and deploy'
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Get yarn cache directory path
+      id: yarn-cache-dir-path
+      run: echo "::set-output name=dir::$(yarn cache dir)"
+    - uses: actions/cache@v2
+      id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+      with:
+        path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+        key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-yarn-
+    - run: yarn --prefer-offline
```

Comment:
We could turn this into a `uses` and point to a local action to stop duplication.